From bf0ac8c527a73ac81376f5497b2ad86605c18fae Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Tue, 23 Jan 2018 23:36:19 +0100
Subject: [PATCH 167/194] Snap: Update pano and video icons to be more like
 photo icons

* Put the colored ring on the _outside_ of the shutter
* While we're at it, cleanup naming and usage of color resources

Change-Id: I4f344fa272374d043f2b83dba7248e263fab8bf0
---
 res/drawable/shutter_button_normal.xml     | 4 ++--
 res/drawable/shutter_button_pressed.xml    | 4 ++--
 res/drawable/shutter_button_stop.xml       | 4 ++--
 res/drawable/shutter_button_video.xml      | 4 ++--
 res/drawable/shutter_button_video_dim.xml  | 4 ++--
 res/drawable/shutter_button_video_stop.xml | 4 ++--
 res/drawable/shutter_panorama.xml          | 4 ++--
 res/drawable/shutter_panorama_dim.xml      | 4 ++--
 res/values/cm_colors.xml                   | 8 ++++----
 9 files changed, 20 insertions(+), 20 deletions(-)

diff --git a/res/drawable/shutter_button_normal.xml b/res/drawable/shutter_button_normal.xml
index 9e1f9aad1..621922c37 100644
--- a/res/drawable/shutter_button_normal.xml
+++ b/res/drawable/shutter_button_normal.xml
@@ -6,11 +6,11 @@
     android:viewportHeight="80">
 
     <path
-        android:fillColor="@color/pic"
+        android:fillColor="@color/shutter"
         android:strokeWidth="1"
         android:pathData="M 40 10.5 C 56.29240012 10.5 69.5 23.70759988 69.5 40 C 69.5 56.29240012 56.29240012 69.5 40 69.5 C 23.70759988 69.5 10.5 56.29240012 10.5 40 C 10.5 23.70759988 23.70759988 10.5 40 10.5 Z" />
     <path
-        android:strokeColor="@color/blue"
+        android:strokeColor="@color/photo"
         android:strokeWidth="7"
         android:pathData="M 40 3.5 C 60.1583933688 3.5 76.5 19.8416066312 76.5 40 C 76.5 60.1583933688 60.1583933688 76.5 40 76.5 C 19.8416066312 76.5 3.5 60.1583933688 3.5 40 C 3.5 19.8416066312 19.8416066312 3.5 40 3.5 Z" />
 
diff --git a/res/drawable/shutter_button_pressed.xml b/res/drawable/shutter_button_pressed.xml
index 32dbd58ee..98d290ff9 100644
--- a/res/drawable/shutter_button_pressed.xml
+++ b/res/drawable/shutter_button_pressed.xml
@@ -6,11 +6,11 @@
     android:viewportHeight="80">
 
     <path
-        android:fillColor="@color/pic_pressed"
+        android:fillColor="@color/shutter_pressed"
         android:strokeWidth="1"
         android:pathData="M 40 12.5 C 55.1878306203 12.5 67.5 24.8121693797 67.5 40 C 67.5 55.1878306203 55.1878306203 67.5 40 67.5 C 24.8121693797 67.5 12.5 55.1878306203 12.5 40 C 12.5 24.8121693797 24.8121693797 12.5 40 12.5 Z" />
     <path
-        android:strokeColor="@color/blue"
+        android:strokeColor="@color/photo"
         android:strokeWidth="7"
         android:pathData="M 40 3.5 C 60.1583933688 3.5 76.5 19.8416066312 76.5 40 C 76.5 60.1583933688 60.1583933688 76.5 40 76.5 C 19.8416066312 76.5 3.5 60.1583933688 3.5 40 C 3.5 19.8416066312 19.8416066312 3.5 40 3.5 Z" />
 
diff --git a/res/drawable/shutter_button_stop.xml b/res/drawable/shutter_button_stop.xml
index 8954092bc..5d23f2578 100644
--- a/res/drawable/shutter_button_stop.xml
+++ b/res/drawable/shutter_button_stop.xml
@@ -6,11 +6,11 @@
     android:viewportHeight="80">
 
     <path
-        android:fillColor="@color/pano_stop"
+        android:fillColor="@color/shutter"
         android:strokeWidth="1"
         android:pathData="M 40 12.5 C 55.1878306203 12.5 67.5 24.8121693797 67.5 40 C 67.5 55.1878306203 55.1878306203 67.5 40 67.5 C 24.8121693797 67.5 12.5 55.1878306203 12.5 40 C 12.5 24.8121693797 24.8121693797 12.5 40 12.5 Z" />
     <path
-        android:strokeColor="@color/white"
+        android:strokeColor="@color/pano_stop"
         android:strokeWidth="7"
         android:pathData="M 40 3.5 C 60.1583933688 3.5 76.5 19.8416066312 76.5 40 C 76.5 60.1583933688 60.1583933688 76.5 40 76.5 C 19.8416066312 76.5 3.5 60.1583933688 3.5 40 C 3.5 19.8416066312 19.8416066312 3.5 40 3.5 Z" />
 
diff --git a/res/drawable/shutter_button_video.xml b/res/drawable/shutter_button_video.xml
index 8c70e7d85..3fb897733 100644
--- a/res/drawable/shutter_button_video.xml
+++ b/res/drawable/shutter_button_video.xml
@@ -6,11 +6,11 @@
     android:viewportHeight="80">
 
     <path
-        android:fillColor="@color/video"
+        android:fillColor="@color/shutter"
         android:strokeWidth="1"
         android:pathData="M 40 10.5 C 56.29240012 10.5 69.5 23.70759988 69.5 40 C 69.5 56.29240012 56.29240012 69.5 40 69.5 C 23.70759988 69.5 10.5 56.29240012 10.5 40 C 10.5 23.70759988 23.70759988 10.5 40 10.5 Z" />
     <path
-        android:strokeColor="@color/white"
+        android:strokeColor="@color/video"
         android:strokeWidth="7"
         android:pathData="M 40 3.5 C 60.1583933688 3.5 76.5 19.8416066312 76.5 40 C 76.5 60.1583933688 60.1583933688 76.5 40 76.5 C 19.8416066312 76.5 3.5 60.1583933688 3.5 40 C 3.5 19.8416066312 19.8416066312 3.5 40 3.5 Z" />
 
diff --git a/res/drawable/shutter_button_video_dim.xml b/res/drawable/shutter_button_video_dim.xml
index c3de0c04c..5e5e35eaa 100644
--- a/res/drawable/shutter_button_video_dim.xml
+++ b/res/drawable/shutter_button_video_dim.xml
@@ -6,11 +6,11 @@
     android:viewportHeight="80">
 
     <path
-        android:fillColor="@color/video_pressed"
+        android:fillColor="@color/shutter_pressed"
         android:strokeWidth="1"
         android:pathData="M 40 12.5 C 55.1878306203 12.5 67.5 24.8121693797 67.5 40 C 67.5 55.1878306203 55.1878306203 67.5 40 67.5 C 24.8121693797 67.5 12.5 55.1878306203 12.5 40 C 12.5 24.8121693797 24.8121693797 12.5 40 12.5 Z" />
     <path
-        android:strokeColor="@color/white"
+        android:strokeColor="@color/video"
         android:strokeWidth="7"
         android:pathData="M 40 3.5 C 60.1583933688 3.5 76.5 19.8416066312 76.5 40 C 76.5 60.1583933688 60.1583933688 76.5 40 76.5 C 19.8416066312 76.5 3.5 60.1583933688 3.5 40 C 3.5 19.8416066312 19.8416066312 3.5 40 3.5 Z" />
 
diff --git a/res/drawable/shutter_button_video_stop.xml b/res/drawable/shutter_button_video_stop.xml
index 9acc1a297..f239f485a 100644
--- a/res/drawable/shutter_button_video_stop.xml
+++ b/res/drawable/shutter_button_video_stop.xml
@@ -6,11 +6,11 @@
     android:viewportHeight="80">
 
     <path
-        android:fillColor="@color/video_stop"
+        android:fillColor="@color/shutter"
         android:strokeWidth="1"
         android:pathData="M 40 12.5 C 55.1878306203 12.5 67.5 24.8121693797 67.5 40 C 67.5 55.1878306203 55.1878306203 67.5 40 67.5 C 24.8121693797 67.5 12.5 55.1878306203 12.5 40 C 12.5 24.8121693797 24.8121693797 12.5 40 12.5 Z" />
     <path
-        android:strokeColor="@color/white"
+        android:strokeColor="@color/video_stop"
         android:strokeWidth="7"
         android:pathData="M 40 3.5 C 60.1583933688 3.5 76.5 19.8416066312 76.5 40 C 76.5 60.1583933688 60.1583933688 76.5 40 76.5 C 19.8416066312 76.5 3.5 60.1583933688 3.5 40 C 3.5 19.8416066312 19.8416066312 3.5 40 3.5 Z" />
 
diff --git a/res/drawable/shutter_panorama.xml b/res/drawable/shutter_panorama.xml
index e68ddc3dc..b5dfe21e1 100644
--- a/res/drawable/shutter_panorama.xml
+++ b/res/drawable/shutter_panorama.xml
@@ -6,11 +6,11 @@
     android:viewportHeight="80">
 
     <path
-        android:fillColor="@color/pano"
+        android:fillColor="@color/shutter"
         android:strokeWidth="1"
         android:pathData="M 40 10.5 C 56.29240012 10.5 69.5 23.70759988 69.5 40 C 69.5 56.29240012 56.29240012 69.5 40 69.5 C 23.70759988 69.5 10.5 56.29240012 10.5 40 C 10.5 23.70759988 23.70759988 10.5 40 10.5 Z" />
     <path
-        android:strokeColor="@color/white"
+        android:strokeColor="@color/pano"
         android:strokeWidth="7"
         android:pathData="M 40 3.5 C 60.1583933688 3.5 76.5 19.8416066312 76.5 40 C 76.5 60.1583933688 60.1583933688 76.5 40 76.5 C 19.8416066312 76.5 3.5 60.1583933688 3.5 40 C 3.5 19.8416066312 19.8416066312 3.5 40 3.5 Z" />
 
diff --git a/res/drawable/shutter_panorama_dim.xml b/res/drawable/shutter_panorama_dim.xml
index a9b69e347..49845b39b 100644
--- a/res/drawable/shutter_panorama_dim.xml
+++ b/res/drawable/shutter_panorama_dim.xml
@@ -6,11 +6,11 @@
     android:viewportHeight="80">
 
     <path
-        android:fillColor="@color/pano_pressed"
+        android:fillColor="@color/shutter_pressed"
         android:strokeWidth="1"
         android:pathData="M 40 12.5 C 55.1878306203 12.5 67.5 24.8121693797 67.5 40 C 67.5 55.1878306203 55.1878306203 67.5 40 67.5 C 24.8121693797 67.5 12.5 55.1878306203 12.5 40 C 12.5 24.8121693797 24.8121693797 12.5 40 12.5 Z" />
     <path
-        android:strokeColor="@color/white"
+        android:strokeColor="@color/pano"
         android:strokeWidth="7"
         android:pathData="M 40 3.5 C 60.1583933688 3.5 76.5 19.8416066312 76.5 40 C 76.5 60.1583933688 60.1583933688 76.5 40 76.5 C 19.8416066312 76.5 3.5 60.1583933688 3.5 40 C 3.5 19.8416066312 19.8416066312 3.5 40 3.5 Z" />
 
diff --git a/res/values/cm_colors.xml b/res/values/cm_colors.xml
index 143e073ae..5331f6ca4 100644
--- a/res/values/cm_colors.xml
+++ b/res/values/cm_colors.xml
@@ -23,14 +23,14 @@
     <color name="green">#4CAF50</color>
     <color name="blue">#2196F3</color>
 
-    <color name="pic">@color/white</color>
-    <color name="pic_pressed">@color/grey</color>
+    <color name="shutter">@color/white</color>
+    <color name="shutter_pressed">@color/grey</color>
+
+    <color name="photo">@color/blue</color>
 
     <color name="video">@color/red</color>
-    <color name="video_pressed">#D32F2F</color>
     <color name="video_stop">#E57373</color>
 
     <color name="pano">@color/green</color>
-    <color name="pano_pressed">#388E3C</color>
     <color name="pano_stop">#81C784</color>
 </resources>
-- 
2.17.1

